web_src/js/features/repo-issue-list.js

```diff
@@ -203,3 +203,15 @@ export function initRepoIssueList() {
   initRepoIssueListAuthorDropdown();
   initIssuePinSort();
 }
+
+export function initHighlightArchivedLabels() {
+  if (!document.querySelectorAll('.show-archived-labels-checkbox').length) return;
+  const showArchivedLabelCheckBox = document.querySelector('.show-archived-labels-checkbox');
+  if (!showArchivedLabelCheckBox) return;
+  document.querySelector('.show-archived-labels-checkbox')
```

Comment:
Querying the same selector 3 times?

![image](https://github.com/go-gitea/gitea/assets/2114189/75d455c3-38cf-4c6a-9f36-35194d05847f)